creator-node/src/AsyncProcessingQueue.ts

```diff
@@ -1,36 +1,51 @@
-const { Queue, Worker } = require('bullmq')
-const { logger: genericLogger } = require('./logging')
-const config = require('./config')
-const redisClient = require('./redis')
-const { clusterUtils } = require('./utils')
+import type { Job } from 'bullmq'
+import type { Request } from 'express'
+import type { SpanContext } from '@opentelemetry/api'
+
+import { Queue, Worker } from 'bullmq'
+import { logger as genericLogger } from './logging'
+import { clusterUtils, ValuesOf } from './utils'
 
 // Processing fns
-const {
-  handleTrackContentRoute: trackContentUpload,
-  handleTranscodeAndSegment: transcodeAndSegment,
-  handleTranscodeHandOff: transcodeHandOff
-} = require('./components/tracks/tracksComponentService')
+import {
+  handleTrackContentRoute as trackContentUpload,
+  handleTranscodeAndSegment as transcodeAndSegment,
+  handleTranscodeHandOff as transcodeHandOff
+} from './components/tracks/tracksComponentService'
+import { instrumentTracing, tracing } from './tracer'
+
 const {
   processTranscodeAndSegments
 } = require('./components/tracks/trackContentUploadManager')
-const { instrumentTracing, tracing } = require('./tracer')
+const redisClient = require('./redis')
+const config = require('./config')
 
 const MAX_CONCURRENCY = 100
 const EXPIRATION_SECONDS = 86400 // 24 hours in seconds
-const PROCESS_NAMES = Object.freeze({
+
+const QUEUE_NAME = 'async-processing'
+
+const ASYNC_PROCESSING_QUEUE_HISTORY = 500
+
+export const ProcessNames = {
   trackContentUpload: 'trackContentUpload',
   transcodeAndSegment: 'transcodeAndSegment',
   processTranscodeAndSegments: 'processTranscodeAndSegments',
   transcodeHandOff: 'transcodeHandOff'
-})
-const PROCESS_STATES = Object.freeze({
+} as const
+
+export const ProcessStates = {
   IN_PROGRESS: 'IN_PROGRESS',
   DONE: 'DONE',
   FAILED: 'FAILED'
-})
-const QUEUE_NAME = 'async-processing'
+} as const
 
-const ASYNC_PROCESSING_QUEUE_HISTORY = 500
+type AddTaskParams = {
+  logContext: { requestID: string }
```

Comment:
i think we should make this type `any` because even though we're only printing the `requestID` here, there may be other fields that are passed in. 

https://github.com/AudiusProject/audius-protocol/pull/4098/files#diff-330e68b54fe69419f61d4adb76b7b255968de4a8b28166059adc53b03fae1136R222

Reply:
also like how here `logContext` is of type `any` https://github.com/AudiusProject/audius-protocol/pull/4098/files#diff-330e68b54fe69419f61d4adb76b7b255968de4a8b28166059adc53b03fae1136R277

Reply:
I wanna avoid making it `any` if possible. Perhaps we can add a `LogContext` type in `utils` somewhere so both `CustomRequest` and `AddTaskParams` can use it